capstone_project/src/main/webapp/pages/comments.jsp

```diff
@@ -48,7 +51,8 @@ limitations under the License.
         </div>
         <div  class="input-field">
           <em class="material-icons prefix">school</em>
-          <select name="schools" id="schools" required>
+          <!-- COME BACK AND MAKE THIS REQUIRED -->
```

Comment:
nit: make this a TODO

Reply:
I addressed the comment and removed it! Good catch.